### PR TITLE
PHP 8.1 update: including extensions php-odbc and pdo_dblib

### PR DESCRIPTION
**Pull request by erikaheidi**
This PR updates the PHP 8.1 build to compile two additional PHP extensions:

- `php-odbc`, via `unixodbc`
- `php-pdo_dblib`, via `freetds`

### CODE REVIEW

**Comment by erikaheidi** [comment]
Maybe I should have sent two separate PRs to avoid issues with CI 😓 I will break these down.

**Comment by erikaheidi** [comment]
> Maybe I should have sent two separate PRs to avoid issues with CI 😓 I will break these down.

All good now!

File: php-8.1.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
     - license: PHP-3.01
   dependencies:
     provides:
-      - php=${{package.version}}-r${{package.epoch}}}
+      - php=${{package.version}}-r${{package.epoch}}
```
Comment (vaikas) [inline php-8.1.yaml:10]: FYI: You can replace this with ${{package.full-version}} that creates this for you :)

**Review by vaikas — DISMISSED** [review]
Thank you!!!

File: php-8.1.yaml (hunk 1 above)
Comment (erikaheidi) [reply php-8.1.yaml]: updated now!

**Review by dlorenc — APPROVED** [review]

**Review by imjasonh — APPROVED** [review]

**Pull request merged**